packages/dashboard/src/components/sidePanel/index.tsx

```diff
@@ -1,29 +1,35 @@
 import React, { FC } from 'react';
 import { Container, Header } from '@cloudscape-design/components';
 import PropertiesAlarmsSection from './sections/propertiesAlarmSection';
-import ThresholdsSection from './sections/thresholdsSection';
-import ChartSettings from './sections/chartSettings';
-import DataSettings from './sections/dataSettings';
 import './index.css';
 import { useSelector } from 'react-redux';
 import { DashboardState } from '../../store/state';
 import { DashboardMessages } from '../../messages';
+import { AppKitComponentTags } from '../../types';
+import TextSettings from './sections/textSettingSection/text';
 
-const SidePanel: FC<{ messageOverrides: DashboardMessages }> = ({
-  messageOverrides: {
-    sidePanel: { propertySection },
-  },
-}) => {
+const SidePanel: FC<{ messageOverrides: DashboardMessages }> = ({ messageOverrides }) => {
   const selectedWidgets = useSelector((state: DashboardState) => state.selectedWidgets);
   if (selectedWidgets.length !== 1) {
     return <div>Currently we only support changing setting for one widget only.</div>;
   }
+
+  const {
+    sidePanel: { propertySection },
+  } = messageOverrides;
+  const selectedWidget = selectedWidgets[0];
+  const isAppKitWidget = AppKitComponentTags.find((tag) => tag === selectedWidget.componentTag);
+  const isTextWidget = selectedWidget.componentTag === 'text';
+
   return (
     <Container header={<Header variant="h3">Configurations</Header>} className={'iot-side-panel'}>
-      <PropertiesAlarmsSection messages={propertySection} />
-      <ThresholdsSection />
-      <ChartSettings />
-      <DataSettings />
+      {/* WIP setting panel UI improvement */}
```

Comment:
are these supposed to be commented out?

Reply:
Yes. I commented these out for ongoing UI adjustments. Will add them back later.

Reply:
updated